orb_models/utils.py

```diff
@@ -0,0 +1,307 @@
+"""Experiment utilities."""
+
+import math
+import os
+import random
+import re
+from collections import defaultdict
+from typing import Dict, List, Mapping, Optional, Tuple, TypeVar
+
+import numpy
+import torch
+import wandb
+from wandb import wandb_run
+
+from orb_models.forcefield import base
+
+T = TypeVar("T")
+
+
+def init_device() -> torch.device:
+    """Initialize a device.
+
+    Initializes a device, making sure to also
```

Comment:
This docstring is now incorrect, also perhaps allow the user to specify the device id